### DISCUSSION TITLE
my project was deployed but url is not working

### DISCUSSION THREAD

**Discussion by manjeerasuggula** [community/community#73652] [New to GitHub]
Hi...All...
I have deployed my website and at the gh-pages area i got the link.. like this.. https://manjeerasuggula.github.io/my-new-website/  
But it's not working...please help me how to resolve my issue..

https://github.com/manjeerasuggula/my-new-website

**Comment by OmBharsakle** [comment]
hellow #20019

**Comment by DJLink** [comment]
you don't seem to have an index.html file in that project root? 
check this help page [https://pages.github.com/#vanilla-step-1](https://pages.github.com/#vanilla-step-1)

**Comment by peterroth** [comment]
The problem is that you didn't create a GH Pages repository, what should be your-username.github.io (yes, that's what your repo must be), and place the website's code there, starting with an index.html
The steps are documented here: https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

**Comment by MegatIrfan** [comment]
Makesure your project file has index.html :)
